js/scripts.js

```diff
@@ -697,17 +678,18 @@ var plugin_formcreator = new function() {
    }
 
    this.addQuestion = function () {
-      var form = $('form[data-itemtype="PluginFormcreatorQuestion"]');
+      var form = document.querySelector('form[data-itemtype="PluginFormcreatorQuestion"]');
       var that = this;
-      $.ajax({
+      $.post({
          url: formcreatorRootDoc + '/ajax/question_add.php',
-         type: "POST",
-         data: form.serializeArray(),
-         dataType: 'json'
+         data: new FormData(form),
+         processData: false,
+         contentType: false,
```

Comment:
processData and contentType are here to help JQuery to work with FormData object; hels to remove useless JQuery use